core/collector.class.inc.php

```diff
@@ -600,16 +606,18 @@ protected function AddHeader($aHeaders)
 	protected function HeaderIsAllowed($sHeader)
 	{
 		foreach ($this->aFields as $aField) {
-			if (in_array($sHeader, $aField['columns'])) return true;
+			if (in_array($sHeader, $aField['columns'])) {
+				return true;
+			}
```

Comment:
Single line if-return should be allowed?

Reply:
nice to have but tricky to configure
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/2780